src/jit/emitxarch.cpp

```diff
@@ -8713,8 +8713,7 @@ BYTE* emitter::emitOutputCV(BYTE* dst, instrDesc* id, size_t code, CnsVal* addc)
             code &= 0x0000FFFF;
         }
 
-        if ((ins == INS_movsx || ins == INS_movzx || ins == INS_cmpxchg || ins == INS_xchg || ins == INS_xadd ||
-             insIsCMOV(ins)) &&
```

Comment:
As far as I can tell this is incorrect, CMOV doesn't have a "w" bit, word sized CMOVs use the operand size prefix. That said, this change isn't required for this PR and if you have concerns with the lack of testing around this I can remove it. It's just something I ran into while playing with if conversion.